README.md

```diff
@@ -118,12 +118,11 @@ Follow along as I use mixins to make our grid system more semantic.
 </div>
 ```
 
-## Lab: Semantic Markup
+## Lab: Refactoring for 'view-state' vs. modals
 
-1.  Using the code I used above in your `index.scss` file refactor on your own.
-1.  Write your own mobile ready page using Bootstrap-Sass classes.
-1.  Refactor your HTML from earlier to be more semantic (copy/paste your
-previous code and refactor it using bootstrap-sass to make it semantic).
+-   In your squads, think about how to take the various modals we have in place and
+think about how you might use different 'view-states' to achieve the same
```

Comment:
It can be a little difficult to think of "modals" as view states.

When @J-Weeks designed this lesson, the point wasn't about the modals, the point was about the semantics of the tags we choose in HTML5, and how using bootstrap sass emphasizes semantics. I'm not sure that knowledge is present in the README, or if it came through when you were reviewing code. Do you have any suggestions?